themes/compose/assets/sass/_base.sass

```diff
@@ -187,10 +188,19 @@ button
       &:hover
         color: var(--light)
 
-      
+
 footer
   min-height: 150px
   a
     color: var(--theme)
     &:hover
       color: var(--light)
+
+.md
+  ul
+    padding-left: 1rem;
+  li
+    font-family: monospace;
```

Comment:
I've always wondered about the use of a monospace font for these items on the front page. Everywhere else we use variable width fonts. I know you didn't change this, but what do you think? :)

Reply:
Yeah I was just copying this from existing styles. I think mnoospace is just silly outside the context of a `pre` or a `code` tag